mp4parse_capi/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "mp4parse_capi"
+version = "0.5.0"
+authors = [
+  "Ralph Giles <giles@mozilla.com>",
+  "Matthew Gregan <kinetik@flim.org>",
+]
+
+description = "Parser for ISO base media file format (mp4)"
+documentation = "https://mp4parse-docs.surge.sh/mp4parse/"
+license = "MPL-2.0"
+
+repository = "https://github.com/mozilla/mp4parse-rust"
+
+# Cargo includes random files from the working directory
+# by default! Avoid bloating the package with test files.
+exclude = [
+  "*.mp4",
+]
+
+build = "build.rs"
+
+[dependencies]
+"mp4parse" = {path = "../mp4parse"}
```

Comment:
```
$ cargo publish
    Updating registry `https://github.com/rust-lang/crates.io-index`
error: all path dependencies must have a version specified when publishing.
dependency `mp4parse` does not specify a version
```

I filed https://github.com/rust-lang/cargo/pull/3060 to add the same check to `cargo package`.